tests/test_core.py

```diff
@@ -1710,6 +1710,8 @@ def test_runtime_stacksave(self):
     self.do_run(src, 'success')
 
   def test_memorygrowth(self):
+    if 'ALLOW_MEMORY_GROWTH' in self.test_mode_settings:
```

Comment:
I think you can already do this with `if self.get_setting('ALLOW_MEMORY_GROWTH')`